conda/cli/conda_argparse.py

```diff
@@ -161,17 +161,24 @@ def error(self, message):
 
         super(ArgumentParser, self).error(message)
 
-    def print_help(self):
-        super(ArgumentParser, self).print_help()
+    def print_usage(self, file=None):
+        if file is None:
+            file = sys.stderr
+        super().print_usage(file)
+
+    def print_help(self, file=None):
+        if file is None:
+            file = sys.stderr
+        super().print_help(file)
 
         if sys.argv[1:] in ([], [''], ['help'], ['-h'], ['--help']):
             from .find_commands import find_commands
             other_commands = find_commands()
             if other_commands:
                 builder = ['']
                 builder.append("conda commands available from other packages:")
-                builder.extend('  %s' % cmd for cmd in sorted(other_commands))
-                print('\n'.join(builder))
+                builder.extend("  %s" % cmd for cmd in sorted(other_commands))
+                self._print_message("\n".join(builder), file)
```

Comment:
It looks like these lines should be executed by the test, but pytest-cov doesn't find it because it's in a subprocess